utils/src/main/java/io/atomix/utils/TimestampPrinter.java

```diff
@@ -33,7 +35,7 @@ public static TimestampPrinter of(long timestamp) {
     return new TimestampPrinter(timestamp);
   }
 
-  private static final SimpleDateFormat FORMAT = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss,SSS");
+  private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss,SSS");
```

Comment:
instead of sync'ing on SimpleDateFormat or creating a new SimpleDateFormat or using threadlocal we can just use the new java.time API in java 8 which is thread safe.